juju-charms/sample_overlay_bundles/bionic-queens-nfs.yaml

```diff
@@ -0,0 +1,96 @@
+variables:
+  openstack-origin: &openstack-origin cloud:bionic-train
```

Comment:
this is an exceptional case which we need to for dmapi only.
opestack-origin is a global variable, so lets avoid using it here.

Reply:
Fixed